content/en/docs/overview/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Overview"
+weight: 200
+---
+
+This chapter will give you an overview of the most important language features of Go.
+Since it is hard to learn a lot of new language features at once, we deliberatly left out certain constructs which, based on our expirience, are not used very often and which are not required most of the time.
```

Comment:
I would try and use short sentences and avoid negatives. Feel free to adjust further.
```suggestion
Our goal is to introduce you to the language. Even though Go is a simple language, there are also more advanced features. We limited the course to language features that we ourselves often use. 

After this course you should be able to write basic Go programs.
```

Reply:
@jyniybinc Tried to simplify the sentence.
I would like to stress that the topics in the overview chapter are enough to understand probably 85% of all Go programs.